part-2-server/server.js

```diff
@@ -0,0 +1,69 @@
+//LISA
+
+var http = require('http');
+var fs = require('fs');
+
+var server = http.createServer(function(request, response) {
+    var url = request.url;
+    console.log(url)
+    if (url === '/') {
+        fs.readFile('index.html', function(err, data) {
+            response.write(data.toString())
+            response.end()
+        })
+    } else if ((url === '/page-2') || (url === '/page-3')) {
+        fs.readFile('page-2.html', function(err, data) {
+            response.write(data.toString())
+            response.end()
+        })
+
+    // // // //   //	I DONT KNOW T_T!!!!
+    // } else if (url === '/poem-' + i) {
+    //     for (var i = 0; i < poem.length; i++) {
+    //         fs.readFile('poem-' + i + 'txt', function(err, data) {
+    //             response.write(data.toString())
+    //             response.end()
+    //         })
+    //     }
+
+        // } else if (url === '/poem-0') {
+        //     fs.readFile('poem-0.txt', function(err, data) {
+        //         response.write(data.toString())
+        //         response.end()
+        //     })
+
```

Comment:
Good thought, but would need to break the url into a substring in order for it to match '/poem-', or else it will never hit this `else if` conditional